### PR TITLE
Feature/bluetooth

### PR DESCRIPTION
**Pull request by kprotopapas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Code Climate: [![Test Coverage](https://api.codeclimate.com/v1/badges/eb499ed5d3
 An Android app that is realized as part of the SDP course at EPFL.
 
 Standalone client for playing Go online against real players and offline against bots. Some ideas of features are:  
- - Interfacing with the OGS website for online playing (most popular Go website)
+ - Interfacing with the OGS website foxr online playing (most popular Go website)
```
Comment (sirpudika) [inline README.md]: Typo?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.github.gogetters.letsgo">
 
+    <uses-permission android:name="android.permission.BLUETOOTH" />
+    <uses-permission android:name="android.permission.BLUETOOTH" />
```
Comment (sirpudika) [inline app/src/main/AndroidManifest.xml]: duplicate

File: app/src/main/java/com/github/gogetters/letsgo/util/BluetoothClient.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.github.gogetters.letsgo.util
+
+class BluetoothClient {
+
```
Comment (sirpudika) [inline app/src/main/java/com/github/gogetters/letsgo/util/BluetoothClient.kt:11]: What is that used for? Comment or fill it

**Review by sirpudika — COMMENTED** [review]
The major part looks great. Just consider my comments on small issues.

File: app/src/main/java/com/github/gogetters/letsgo/util/BluetoothGTPService.kt [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+    private lateinit var handleThread: ConnectedThread
+
+    fun connect(socket: BluetoothSocket) {
+        handleThread = ConnectedThread(socket)
+        handleThread.run()
+    }
+
+    fun parseCommand(command: GTPCommand) {
+
```
Comment (whitewind664) [inline app/src/main/java/com/github/gogetters/letsgo/util/BluetoothGTPService.kt:11]: maybe put a todo comment

**Review by lupusmalus — APPROVED** [review]

**Review by sirpudika — APPROVED** [review]
Very nice what you did. The only bad thing is that it's a lot of untested code - but yea, we talked about that. (And I really see that testing it is hard)

**Review by sirpudika — COMMENTED** [review]
It's cool what you did, but it will break some of my code for the tutorial game..... (and doesn't build at the moment). Will need some work before merging to main. In my opinion, at least the game parts should be tested (or at least tried to).

**Pull request merged**